config/kernel-fpu.m4

```diff
@@ -1,9 +1,12 @@
-dnl # 
+dnl #
 dnl # Handle differences in kernel FPU code.
 dnl #
 dnl # Kernel
 dnl # 5.16:	XCR code put into asm/fpu/xcr.h
-dnl # 		HAVE_KERNEL_FPU_XCR_HEADER
+dnl #		HAVE_KERNEL_FPU_XCR_HEADER
+dnl #
+dnl #		XSTATE_XSAVE and XSTATE_XRESTORE aren't accessible any more
+dnl #		HAVE_KERNEL_FPU_INTERNAL_NO_XSTATE
```

Comment:
This isn't defined anywhere, shouldn't this be `HAVE_KERNEL_FPU_XSAVE_INTERNAL`?

Reply:
Yes, will fix.